src/App.scss

```
@@ -0,0 +1,44 @@
+@use 'common/Button/Button.module.scss';
+@use 'common/Input/Input.module.scss';
+
+@use 'components/CourseInfo/CourseInfo.module.scss';
+@use 'components/Header/Header.module.scss';
+@use'components/Courses/components/SearchBar/SearchBar.module.scss';
+@use'components/Courses/components/CourseCard/CourseCard.module.scss';
+@use'components/Courses/Courses.module.scss';
+@use'components/EmptyCourseList/EmptyCourseList.module.scss';
+* {
```

In most cases `*` is used only for changing `box-sizing` css property, so I would suggest to change it to


```suggestion
*,
*::after
*::before {
  box-sizing: border-box;
}

body {
  font-family: Open Sans;
  font-weight: 400;
  line-height: 1.5rem;
  font-size: 1rem;
  color: #333E48;
}
```